lib/tasks/data_updates.rake

```diff
@@ -4,8 +4,15 @@ namespace :data_updates do
     if Rails.env.development?
       Rake::Task["data_updates:run"].execute
     else
-      # Ensure new code has been deployed before we run our update scripts
-      DataUpdateWorker.perform_in(10.minutes)
+      # @mstruve: Due to many folks already hosting their Forems on Heroku lets
+      # set a friendly default for them. Ideally we get everyone to use
+      # the ENV variable below but until we can do a hard cut over with
+      # a version I think this is a good compromise
+      default_delay = ENV["HEROKU_SLUG_COMMIT"].present? ? 10.minutes : 0
+
+      # Use the env variable to delay running data update scripts if your
+      # deploy strategy requires it
+      DataUpdateWorker.perform_in(ENV["DATA_UPDATE_WORKER_DELAY_SECONDS"] || default_delay)
```

Comment:
Just a tiny cosmetic note, can we namespace the variables so they are easier to find? We have many of them and this might be just one or it might be one of many, so I'm proposing, instead of `DATA_UPDATE_WORKER_DELAY_SECONDS`, we use `WORKERS_DATA_UPDATE_DELAY_SECONDS`, so that we need a new one for another worker, they'll nicely sort in the `heroku` command line list as:

```
WORKERS_DATA_UPDATE_DELAY_SECONDS
WORKERS_EMAIL_ABCDE_DELAY_SECONDS
```

Definitely non blocking :)